Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Messagebox/Classes/InternalKryptonMessageBoxExtended.cs

```diff
@@ -2053,22 +2053,22 @@ private void ShowCopyButton(bool showCopyButton, string copyButtonText)
             switch (customButtonOptions)
             {
                 case ExtendedMessageBoxCustomButtonOptions.ONEBUTTON:
-                    _button1.Text = buttonOneText;
+                    _button3.Text = buttonOneText;
```

Comment:
If its "OneButton", then why set button3 to buttonOneText ?

Reply:
It's a bit weird, I thought so to, but upon testing I found that all text relating to one button only needs to be on `_button3`.

Reply:
Then comment this in the code please ?

Reply:
Done, also removed `KryptonMessageBoxExtended` due to the previously mentioned reason.